src/View3d.ts

```diff
@@ -232,6 +232,20 @@ export class View3d {
     this.redraw();
   }
 
+  setShowBoundingBox(volume: Volume, showBoundingBox: boolean): void {
+    if (this.image) {
+      this.image.setShowBoundingBox(showBoundingBox);
+    }
+    this.redraw();
+  }
+
+  setBoundingBoxColor(volume: Volume, color: [number, number, number]): void {
+    if (this.image) {
+      this.image.setBoundingBoxColor(color);
+    }
+    this.redraw();
+  }
```

Comment:
It was an attempt to distinguish between api calls that modify some aspect of the volume being displayed versus parameters global to the viewer/rendering system.  Plus: In case there are ever multiple volumes loaded at the same time, this would also be used in that scenario.

This whole View3d public interface needs to be looked at closely and brainstormed for something cleaner/simpler/more obvious.